Fabric.Identity.API/Management/UserLoginManager.cs

```diff
@@ -113,7 +113,17 @@ private List<Claim> FilterClaims(IEnumerable<Claim> incomingClaims)
                 // copy the claim as-is
                 else
                 {
-                    filtered.Add(claim);
+                    if (claim.Type == FabricIdentityConstants.FabricClaimTypes.Groups)

```

Comment:
Should we add some of these nested if's to an else if to reduce some nesting?

Reply:
Do you mean pulling the `if (claim.Type == FabricIdentityConstants.FabricClaimTypes.Groups)` out of the `else`?

Reply:
Yeah, basically combining it with the else above to make it an else if - but its not really necessary.